wiki/Guides/How_to_Use_the_Offset_Wizard/en.md

```diff
@@ -1,3 +1,25 @@
 # How to use the offset wizard
+*See also: [Universal Offset](/wiki/Universal_offset)*
 
-This guide was incorrect and has been removed.
+If the [Hitsounds](/wiki/Beatmapping/Hitsound) feel out of sync with the music, in every beatmap you play, you will more than likely need to change your [universal offset](/wiki/Universal_offset). The easiest way to do this is by using the **offset wizard**.
+
+Follow these steps to adjust your universal offset using the offset wizard:
+
+**Opening the offset wizard**:
+
+1. Open osu!.
+2. Go to the `Options` menu, or press `Ctrl` + `O`.
+3. Search "offset" in the quick-search and click on `Offset wizard`.
+
+The offset wizard will then open, playing the song that was previously playing in osu!, while a green bar scrolls across the screen and a metronome plays.
+
+**Using the offset wizard to calibrate your universal offset**:
```

Comment:
```suggestion
**Using the Offset wizard to calibrate your Universal offset**:
```

Reply:
If you're gonna capitalise it, capitalise as `Universal Offset`, not just half.